src/js/components/replenishment/CreateReplenishment.jsx

```diff
@@ -101,6 +125,14 @@ function validate(values) {
       errors.requirements[key] = { quantity: 'react.replenishment.error.quantity.label' };
     }
   });
+  if (numberOfProductsSelected(values.requirements) < 1) {
```

Comment:
This can be as simple as: 
```
const anyRowSelected = _.find(values.requirements, row => row.checked);
if (!anyRowSelected) {
    ...
}
```
(or if you need an `anyRowSelected` helper function for other places, then do something similar as suggested in the comment on line 331)